pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -612,6 +757,39 @@ public List<KitRequestShipping> getKitsByHruid(String hruid) {
         return kitRequestList;
     }
 
+    public void updateKitRequestDDPLabel(int dsmKitRequestId, String ddpLabel) {
```

Comment:
I am not able to find the usage of this method, but in general I can't think of when we are changing the ddpLabel of a kit. It is one of the unique ids of the kit

Reply:
Gaah, I forgot to remove this--was a leftover from an earlier approach.  Will delete.